roles/StackStorm.nodejs/defaults/main.yml

```
@@ -1,3 +1,3 @@
 ---
 # nodejs version to install
-nodejs_major_version: "10"
+nodejs_major_version: "{% if st2_version == 'latest' %}14{% elif st2_version is version('3.5.0', '<') %}10{% else %}14{% endif %}"
```

```suggestion
nodejs_major_version: "{% if st2_version == 'latest' or st2_version is version('3.5.0', '>=') %}14{% else %}10`{% endif %}"
```
What about something like this?

---

Good catch! I tried it with a different order and ran into the issue that version() does not expect a string. Changed it as per your suggestion.